clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/task/datawriter/AbstractPartitionWriter.java

```diff
@@ -364,6 +364,7 @@ private VeniceWriter<byte[], byte[], byte[]> createBasicVeniceWriter() {
             .setChunkingEnabled(chunkingEnabled)
             .setRmdChunkingEnabled(rmdChunkingEnabled)
             .setTime(SystemTime.INSTANCE)
+            .setPartitionCount(getPartitionCount())
```

Comment:
is this line to remove the extra call?

Reply:
Yeah. https://github.com/linkedin/venice/blob/main/internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriter.java#L341-L347